python/pyarrow/_csv.pyx

```diff
@@ -1337,6 +1355,17 @@ cdef class WriteOptions(_Weakrefable):
     def delimiter(self, value):
         deref(self.options).delimiter = _single_char(value)
 
+    @property
+    def quoting_style(self):
+        """
+        Quoting style for CSV writing.
```

Comment:
We would rather reuse the same description as in the `WriteOptions` docstring above.

Reply:
Done